Documentation/MigrationGuide.md

```diff
@@ -1,46 +1,69 @@
 
-- `enumName`: has been replaced by `param.enumName`, should provide default value.
-- `images`: just old, `catalogs` contains the structured information.
+You can also take advantage of that new `--param` feature to make your own templates more customizable, by allowing users to provide arbitrary values via the command line, e.g. using `{{param.foo|default:"Foo"}}` and `{{param.bar|default:"-"}}` in your templates to let users provide custom values using `--param foo=MyFoo --param bar=_`. Just don't forget to document the available params somewhere to let the users of your templates know about those.
 
-#### Storyboards
+### Colors
 
-- `extraImports`: replaced by `modules` (https://github.com/AliSoftware/SwiftGen/pull/243)
-- `sceneEnumName`: has been replaced by `param.sceneEnumName`, should provide default value.
-- `segueEnumName`: has been replaced by `param.segueEnumName`, should provide default value.
-- For each `scene`:
-  - `isBaseViewController`: removed. You can replace it with a test for `baseType == "ViewController"`.
+_📖 see the full context structure [in the documentation here](https://github.com/SwiftGen/SwiftGenKit/blob/master/Documentation/Colors.md)._
 
-#### Strings
+- `enumName`: This variable has been replaced by `param.enumName` — see above.
+- `colors`: this key has been replaced by the `palettes` array, each entry having a `name` and a `colors` property.
+- for each `color`:
+  - `rgb` and `rgba`: these keys have been removed, as they can be composed from the other components (e.g. `#{{color.red}}{{color.green}}{{color.blue}}{{color.alpha}}`).
 
-- `enumName`: has been replaced by `param.enumName`, should provide default value.
-- `strings` and `structuredStrings`: replaced by `tables` array, where each table has a structured `levels` property.
-- `tableName`: replaced by `tables` array, where each table has a `name` property.
+### Fonts
+
+_📖 see the full context structure [in the documentation here](https://github.com/SwiftGen/SwiftGenKit/blob/master/Documentation/Fonts.md)._
+
+- `enumName`: This variable has been replaced by `param.enumName` — see above.
+- for each `font`:
+  - `fontName`: this key has been replaced by the `name` property.
+
+### XCAssets (formerly Images)
+
+_📖 see the full context structure [in the documentation here](https://github.com/SwiftGen/SwiftGenKit/blob/master/Documentation/Assets.md)._
+
+- `enumName`: This variable has been replaced by `param.enumName` — see above.
+- `images`: this key is deprecated. The new root key is named `catalogs` and contains the structured information.
+
+### Storyboards
+
+_📖 see the full context structure [in the documentation here](https://github.com/SwiftGen/SwiftGenKit/blob/master/Documentation/Storyboards.md)._
+
+- `extraImports`: this key has been renamed `modules` (see [SwiftGen/SwftGen#243](https://github.com/AliSoftware/SwiftGen/pull/243))
```

Comment:
Why not use Github's automatic display of its links?
`https://github.com/SwiftGen/SwiftGen/pull/243` -> https://github.com/SwiftGen/SwiftGen/pull/243
Also, this references `AliSoftware` instead of `SwiftGen`.